README.md

```diff
@@ -1,17 +1,15 @@
+# Ikarus
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/IkarusRepo/Ikarus/pullRequest?prid=7699962)
```suggestion
# Ikarus

```